ui/app/AppLayouts/Profile/stores/ProfileStore.qml

```diff
@@ -134,10 +134,6 @@ QtObject {
         root.profileModule.moveLink(fromRow, toRow)
     }
 
-    function resetSocialLinks() {
```

Comment:
You can remove the whole section afaics, lines 116-144

Reply:
Good point. Indeed there was a comment. I checked all those functions and they are never used. Removed